documentation/modules/operators/con-topic-operator-store.adoc

```diff
@@ -0,0 +1,117 @@
+Updates from operations applied to the local in-memory cache are persisted to a backup topic store on disk.
+The topic store is continually synchronized with updates from Kafka topics or Kubernetes `KafkaTopic` custom resources.
+Operations are handled rapidly with the topic store set up this way,
+but should the in-memory cache crash it is automatically repopulated from the persistent storage.
+
+== Internal topic store topics
+
+Internal topics support the handling of topic metadata.
+
+`__strimzi_store_topic`:: Input topic for storing the topic metadata
+`__strimzi-topic-operator-kstreams-topic-store-changelog`:: Retains a log of compacted topic store values
+
+IMPORTANT: Do not delete these topics, as they are essential to the running of the Topic Operator.
+
+== Migrating topic metadata from ZooKeeper
+
+In previous releases of Strimzi, topic metadata was stored in ZooKeeper.
+The new process removes this requirement, bringing the metadata into the Kafka cluster, and under the control of the Topic Operator.
+
+When upgrading to Strimzi {ProductVersion}, the transition to Topic Operator control of the topic store is seamless.
+Metadata is found and migrated from ZooKeeper, and the old store is cleansed.
+
+== Downgrading to a Strimzi version that uses ZooKeeper to store topic metadata
+
+If you are reverting back to a version of Strimzi earlier than 0.22, which uses ZooKeeper for the storage of topic metadata,
+you still downgrade your Cluster Operator to the previous version,
+then downgrade Kafka brokers and client applications to the previous Kafka version as standard.
+
+However, you must also delete the topics created for the topic store using a `kafka-admin` command, specifying the bootstrap address of the Kafka cluster.
+For example:
+
+[source,shell]
+----
+kubectl run kafka-admin -ti --image={DockerKafkaImageCurrent} --rm=true --restart=Never -- ./bin/kafka-topics.sh --bootstrap-server localhost:9092 --topic __strimzi-topic-operator-kstreams-topic-store-changelog --delete && ./bin/kafka-topics.sh --bootstrap-server localhost:9092 --topic __strimzi_store_topic --delete
+----
+
+The command must correspond to the type of listener and authentication used to access the Kafka cluster.
+
+The Topic Operator will reconstruct the ZooKeeper topic metadata from the state of the topics in Kafka.
+
+== Topic Operator topic replication and scaling
+
+The recommended configuration for the Topic Operator topics is to have a topic replication factor of 3, and a minimum of 2 in-sync replicas.
+
+[source,shell,subs="+quotes,attributes"]
+----
+apiVersion: {KafkaTopicApiVersion}
+kind: KafkaTopic
+metadata:
+  name: my-topic
+  labels:
+    strimzi.io/cluster: my-cluster
+spec:
+  partitions: 1
+  replicas: 3 <1>
+  config:
+    min.insync.replicas=2 <2>
+  #...
+----
+<1> The number of replica topics. Currently, this cannot be changed in the `KafkaTopic` resource, but it can be changed using the `kafka-reassign-partitions.sh` tool.
```

Comment:
```suggestion
<1> The number of partition replicas. Currently, this cannot be changed in the `KafkaTopic` resource, but it can be changed using the `kafka-reassign-partitions.sh` tool.
```